spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubMessageChannelBinder.java

```diff
@@ -48,21 +49,20 @@
 
 	private final PubSubTemplate pubSubTemplate;
 
-	private final PubSubExtendedBindingProperties pubSubExtendedBindingProperties =
-			new PubSubExtendedBindingProperties();
+	private final PubSubExtendedBindingProperties pubSubExtendedBindingProperties;
 
 	public PubSubMessageChannelBinder(String[] headersToEmbed,
-			PubSubChannelProvisioner provisioningProvider, PubSubTemplate pubSubTemplate) {
-
+			PubSubChannelProvisioner provisioningProvider, PubSubTemplate pubSubTemplate,
+			PubSubExtendedBindingProperties pubSubExtendedBindingProperties) {
 		super(headersToEmbed, provisioningProvider);
 		this.pubSubTemplate = pubSubTemplate;
+		this.pubSubExtendedBindingProperties = pubSubExtendedBindingProperties;
 	}
 
 	@Override
 	protected MessageHandler createProducerMessageHandler(ProducerDestination destination,
 			ExtendedProducerProperties<PubSubProducerProperties> producerProperties,
 			MessageChannel errorChannel) {
-
```

Comment:
When method specification is wrapped to several lines, I prefer to have a blank line before method body for better readability. Although I heard that Boot team prefer do not have blank lines in the method body at all...

Reply:
I see. Sounds good, we can do it your way. I'm fine with either; but keeping it consistent will probably be best.